frontend/src/components/App/FishbowlInfo/index.tsx

```diff
@@ -60,7 +62,9 @@ const FishbowlInfo: React.FC<Props> = ({ data }) => {
         <Description>
           <p className="body-sm medium description__title">{data.name}</p>
           <p className="info-text body-xs">{data.description}</p>
-          <ButtonCopyUrl variant="secondary" size="medium" fid={data.slug} locale={data.locale} />
+          {data.slug && (
+            <ButtonCopyUrl variant="secondary" size="medium" fid={data.slug} locale={data.locale} />
```

Comment:
Either the buttonCopyUrl accepts fid as undefined or we have to do this kind of this everytime we use this component. @eddsaura

Reply:
fid should be optional.
- fishbowl list case : You send the fid for each fishbowl
- The rest of cases: you take it directly from next router

Reply:
I think it shouldn't be optional from api-platform. It is the only way to enter the fishbowl right?

Reply:
https://github.com/Stooa/Stooa/blob/main/frontend/src/types/api-platform/interfaces/fishbowl.ts#L20

According to api platform, it is optional. If you remove that ? from the line, we do not need this change (same for other changes related to slug)

Reply:
slug is optional in the database and can be null

Reply:
Reverted this change.